### PR TITLE
Setup scheme for profiling #trivial

### PR DESCRIPTION
**Pull request by marcelofabri**
* [x] I've started my branch from the `develop` branch (gitflow)
  * [x] And I've selected `develop` as the "base" branch for this Pull Request I'm about to create
* [x] I've added an entry in the `CHANGELOG.md` file to explain my changes and credit myself  
      _(or added `#trivial` to the PR title if I think it doesn't warrant a mention in the CHANGELOG)_
  * [ ] Add the entry in the appropriate section (Bug Fix / New Feature / Breaking Change / Internal Change)
  * [ ] Add a period and 2 spaces at the end of your short entry description
  * [ ] Add links to your GitHub profile to credit yourself and to the related PR and issue after your description

---

This allows using Instruments to measure performance

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

Seems like everything is in order 👍 You did a good job here! 🤝

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme [hunk 1]
```diff
@@ -40,8 +40,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "NO"
-      codeCoverageEnabled = "YES">
+      shouldUseLaunchSchemeArgsEnv = "NO">
```
Comment (AliSoftware) [inline SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme]: Needs rebase after the merge of #703, in order to re-include the code coverage in there and not re-remove it I guess?

**Review by AliSoftware — APPROVED** [review]
Be sure to double-check line 43 before merging, probably a conflicting merge after #703.
Otherwise, LGTM 👍

File: SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme (hunk 1 above)
Comment (marcelofabri) [reply SwiftGen.xcodeproj/xcshareddata/xcschemes/swiftgen.xcscheme]: 🤦‍♂️

**Pull request merged**